DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -163,4 +124,37 @@
   <class name="reco::ClusterRemovalInfo::SiPixelClusterRefProd" />
   <class name="reco::ClusterRemovalInfo::SiStripClusterRefProd" />
   <class name="edm::Wrapper<reco::ClusterRemovalInfo>" />
+
+  // fastsim containers
+  
+  <class name="FastTrackerRecHit" ClassVersion="10">
```

Comment:
do we care about any old(er) fastsim files with these hits stored? (or are the hits usually not kept?)